FirebaseInAppMessaging/Sources/Analytics/FIRIAMClearcutLogStorage.m

```diff
@@ -166,10 +166,23 @@ - (void)loadFromCachePath:(NSString *)cacheFilePath {
   NSString *filePath = cacheFilePath == nil ? [self.class determineCacheFilePath] : cacheFilePath;
 
   NSTimeInterval start = [self.timeFetcher currentTimestampInSeconds];
+  id fetchedClearcutRetryRecords;
+  NSData *data = [NSData dataWithContentsOfFile:filePath];
+  if (data) {
+    if (@available(macOS 10.13, iOS 11.0, tvOS 11.0, *)) {
+      fetchedClearcutRetryRecords = [NSKeyedUnarchiver
+          unarchivedObjectOfClasses:[NSSet setWithObjects:[FIRIAMClearcutLogRecord class],
+                                                          [NSArray class], nil]
```

Comment:
nit: I think we should use NSMutableArray to make it extra clear that we are decoding a mutable type. 
```suggestion
                                                          [NSMutableArray class], nil]
```

Reply:
No worries-- thanks!